tests/python/relax/frontend/test_onnx_frontend.py

```diff
@@ -1337,14 +1341,21 @@ def verify_pad_constant_value(constant_value):
     verify_pad_constant_value("")
 
 
-@pytest.mark.skip
-def test_split():
+@pytest.mark.parametrize(
+    "fp_arith, dynamic",
+    [
+        (np.float16, True),
+        (np.float32, True),
+        (np.float16, False),
+        (np.float32, False),
+    ],
+)
+def test_split(fp_arith, dynamic):
     """test_split"""
 
     def verify_split(indata, outdatas, split, axis=0, pass_split=True, opset=11):
-        indata = np.array(indata).astype(np.float32)
-        outdatas = [np.array(o).astype(np.float32) for o in outdatas]
-        inputs = [helper.make_tensor_value_info("input", TensorProto.FLOAT, list(indata.shape))]
+        indata = np.array(indata).astype(fp_arith)
+        outdatas = [np.array(o).astype(fp_arith) for o in outdatas]
```

Comment:
I know this is copy-pasted from the Relay test, but the structure of this test is a bit weird to me. Do you know why we need to give it output data when we test correctness against an ONNX reference? I would think we could refactor this just to give input and output shapes rather than data explicitly.

Reply:
You're right -- there's no need to pass the output data. 

I changed the tests to pass the input and output shapes instead. 

Currently, we can not instantiate inputs with dynamic shapes, so that stays in the verify method of the test for now.